project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject dawran6/emoji "0.1.5"
+(defproject dawran6/emoji "0.1.6"
```

Comment:
[Request] Would you undo the versioning change?  I'll be bumping the minor version and deploy a new version to clojars once this PR is merged.  Thank you!